src/main/java/com/blamejared/controlling/client/gui/GuiNewKeyBindingList.java

```diff
@@ -230,8 +243,9 @@ public void drawEntry(int slotIndex, int x, int y, int listWidth, int slotHeight
                 prefix = EnumChatFormatting.YELLOW + "> " + EnumChatFormatting.RESET + EnumChatFormatting.UNDERLINE;
                 suffix = EnumChatFormatting.RESET.toString() + EnumChatFormatting.YELLOW + " <";
             } else if (hasConflict) {
-                prefix = EnumChatFormatting.RED + "[ " + EnumChatFormatting.RESET;
-                suffix = EnumChatFormatting.RED + " ]";
+                EnumChatFormatting clr = modConflict ? EnumChatFormatting.GOLD : EnumChatFormatting.RED;
+                prefix = clr + "[ " + EnumChatFormatting.RESET;
+                suffix = clr + " ]";
             }
```

Comment:
IMO, if a key binding uses the same key, it should be displayed in red regardless of modifiers.
```suggestion
            } else if (hasConflict) {
                prefix = EnumChatFormatting.RED + "[ " + EnumChatFormatting.RESET;
                suffix = EnumChatFormatting.RED + " ]";
            } else if (modConflict) {
                prefix = EnumChatFormatting.GOLD + "[ " + EnumChatFormatting.RESET;
                suffix = EnumChatFormatting.GOLD + " ]";
            }
```

Reply:
Its more of a soft conflict. e binds wont interfere with ctrl+e binds

Reply:
Also, you have the wrong logic in your suggestion

Reply:
hasConflict will always be true if modConflict is true